### PR TITLE
Init and implementation of RequestHandler()

### PR DESCRIPTION
**Pull request by AriaZhou**

### CODE REVIEW

**Review by Jazz86 — APPROVED** [review]
LGTM

File: src/include/RequestHandler.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#include "HttpResponsePacket.h"
+
+// Basic RequestHandler, father of business handlers
+class RequestHandler {
+public:
+  RequestHandler();
+
+public:
+  // Business processing
+  virtual void BusinessProcessing(const HttpRequestPacket& http_request_packet,
```
Comment (john0312) [inline src/include/RequestHandler.h]: Consider calling this `HandleRequest()` or `ProcessBusinessLogic()`.

File: src/include/RequestHandler.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#define _REQUESTHANDLER_H
+
+#include "HttpHandler.h"
+#include "HttpRequestPacket.h"
+#include "HttpResponsePacket.h"
+
+// Basic RequestHandler, father of business handlers
+class RequestHandler {
+public:
+  RequestHandler();
```
Comment (john0312) [inline src/include/RequestHandler.h]: Maybe set this to default so you don't have to implement it.
`RequestHandler() = default;`

**Review by john0312 — APPROVED** [review]
Approved assuming reviews are addressed.

**Pull request merged**